Quantori.AllureTestOpsClient/Model/Link.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Quantori.AllureTestOpsClient.Model
+{
+    public class Link
+    {
+        public string Name { get; set; }
+        public string Url { get; set; }
+        public string Type { get; set; }
```

Comment:
What is the "Link type"? If the amount of types is known and limited to a reasonable extent, it makes sense to use enum

Reply:
Let's discuss it offline. I am not sure that we are on the same page.